src/modules/job-list/job_data.c

```diff
@@ -112,6 +112,8 @@ static int parse_jobspec_job_name (struct job *job,
             return -1;
         }
     }
+    else
```

Comment:
Commit message: The `/` is probably not needed along with "or" :stuck_out_tongue_winking_eye: